po/de.po

```diff
@@ -19131,19 +19113,16 @@ msgstr ""
 "die Option '--force' zu verwenden.\n"
 
 #: builtin/push.c:294
-#, fuzzy
 msgid ""
 "Updates were rejected because the tip of the remote-tracking\n"
 "branch has been updated since the last checkout. You may want\n"
 "to integrate those changes locally (e.g., 'git pull ...')\n"
 "before forcing an update.\n"
 msgstr ""
 "Aktualisierungen wurden zurückgewiesen, weil die Spitze Ihres aktuellen\n"
-"Branches hinter seinem externen Gegenstück zurückgefallen ist. Führen Sie\n"
-"die externen Änderungen zusammen (z. B. 'git pull ...') bevor Sie \"push\"\n"
-"erneut ausführen.\n"
-"Siehe auch die Sektion 'Note about fast-forwards' in 'git push --help'\n"
-"für weitere Details."
+"Branches hinter seinem externen Gegenstück zurückgefallen ist. Sie sollten\n"
+"die externen Änderungen zusammenführen (z. B. 'git pull ...') bevor Sie die\n"
```

Comment:
Aktualisierungen wurden zurückgewiesen, weil die Spitze des Remote-Tracking-Branches seit dem letzten Checkout aktualisiert wurde. Sie möchten vielleicht diese Änderungen lokal integrieren (z.B. 'git pull ...'), bevor Sie die Aktualisierung erzwingen.

Reply:
Mit dem Abschnitt habe ich mich schwer getan.. "You may want to" ist für mich eher eine Anweisung bzw. ein Vorschlag. Ich hatte diese Meldung selbst schon oft im Englischen vor mir gehabt und weiß, dass man genau in diesem Fall kein "force push" machen sollte.. Deshalb würde ich es gern als Vorschlag formulieren.

Reply:
Ok, mit dem "vielleicht" darin gefällt es mir doch ganz gut, wie du es vorgeschlagen hast.. Alternativen wären "Sie können ..." oder "Sie sollten vielleicht ..."